src/rpc/methods/eth/utils.rs

```diff
@@ -0,0 +1,165 @@
+// Copyright 2019-2025 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use super::types::{EthAddress, EthBytes};
+use crate::rpc::state::{MessageTrace, ReturnTrace};
+use crate::shim::address::Address as FilecoinAddress;
+use crate::shim::state_tree::StateTree;
+
+use anyhow::{bail, Result};
+use cbor4ii::core::dec::Decode as _;
+use cbor4ii::core::Value;
+use fvm_ipld_blockstore::Blockstore;
+use fvm_ipld_encoding::{RawBytes, CBOR, DAG_CBOR, IPLD_RAW};
+use serde::de;
+
+/// The Identity multicodec code.
+pub const IDENTITY: u64 = 0x0;
```

Comment:
It's not available anywhere to import?

Reply:
Unfortunately no. However, we could consider adding it to [fvm_ipld_encoding](https://github.com/filecoin-project/ref-fvm/blob/master/ipld/encoding/src/lib.rs).

Reply:
What about https://github.com/filecoin-project/ref-fvm/blob/e9eef91d173d2cc2cb464c7f988fa2a6cc82577b/shared/src/lib.rs#L59-L60?

Reply:
Thanks!